model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -5,10 +5,13 @@ import org.broadinstitute.dsde.rawls.model.Attributable.AttributeMap
 import org.broadinstitute.dsde.rawls.model.SortDirections.SortDirection
 import org.broadinstitute.dsde.rawls.model.WorkspaceAccessLevels.WorkspaceAccessLevel
 import org.joda.time.DateTime
+import com.netaporter.uri.Uri.parse
```

Comment:
is there a specific reason you chose this instead of `spray.http.Uri`? (even though we know Spray will be going away soonish)

Reply:
The Spray API seemed a bit clunkier: for example, I do not see a way to index to an arbitrary position in the path, only head/tail. And to get host, it's `parsedUri.authority.host` instead of `parsedUri.host`.  But it's certainly functional if you'd rather not introduce a new dependency.